test_board.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Lyngk Website</title>
+
+  <link rel="stylesheet" href="./style.css">
+
+</head>
+
+<body>
+  <html>
+    <body>
+      <div id="unclaimed_label">
+        <h3>
+          Unclaimed colors:
+        </h3>
+      </div>
+      <div id ='summaryandmoves'>
+        <h1>Summary of Lyngk Board 191</h1>

```

Comment:
For consistency with the existing PHP games, let's write: "Lyngk game 191" instead of "Summary of...".  See for example:

http://www.gamerz.net/pbmserv/Zertz/Zertz.php?2000